### DISCUSSION TITLE
MS Teams Webhook

### DISCUSSION THREAD

**Discussion by stavros-k** [outline/outline#5627] [Self Hosting] [answered]
Hello, I'm trying to setup webhook for MS Teams.
I've created an Incoming Webhook in a MS Teams Channel, 
Created a Webhook in Outline pasting the link I've got from the webhook creating in teams, but it does not post anything (selected all events)
It goes to "Disabled" cause of 25+ failures.

Do I have to append anything at the end of the webhook url?
Is MS Teams require something custom that outline needs to add support for?
Thanks!

**Comment by tommoor** [comment, marked as answer]
> Is MS Teams require something custom that outline needs to add support for?

Yes, unfortunately this isn't how webhooks work – and we would need to build a specific integration for MS Teams. It's on the roadmap but I can't speak to a target date

**Reply by stavros-k** [reply]
Thanks for replying!
I'll wait patiently then 😀

**Comment by thomaskrampe** [comment]
The last answer is now almost two years ago, is there anything new from the Roqdmap since a webhook support that works with MS teams is still missing?

**Reply by tommoor** [reply]
Still on the roadmap, but honestly it just doesn't come up very often in comparison to other requests

**Comment by ghost** [comment]
@tommoor, I’ve noticed that user comments are being deleted—can you clarify why?

If the goal is to avoid reminders that feature requests have remained unaddressed for over four years, there are likely more constructive ways to handle that than removing user comments that bring attention to it.

This section is for discussions, not just issue tracking. Are comments being moderated based on content, particularly when users express frustration over long-standing feature requests that have seen little progress? Transparency on this would be appreciated.

**Reply by tommoor** [reply]
> Are comments being moderated based on content, particularly when users express frustration over long-standing feature requests that have seen little progress?

Yes, this is an environment where comments will be moderated. Comments that do not contribute constructively to the conversation will be hidden or deleted.
